src/components/typography/typography.styles.ts

```diff
@@ -0,0 +1,337 @@
+export const typographySx = {
+  "h4": {
+    color: "var(--text-secondary)",
+    fontSize: "0.75rem",
+    fontWeight: "400",
+  },
+  "subtitle2": {
+    color: "var(--text-primary)",
+    fontSize: "0.75rem",
+  },
+  "regular-displayLG": {
```

Comment:
Why variant key use dashed style? 
Can we just use camelCase as normal JS object key convention?